### PR TITLE
Info modal dom

### PR DESCRIPTION
**Pull request by viktornoskire**
# Info modal DOM
This is where I built the Document Object Model with JavaScript and the JSON file with data. 

## Added 
- Title and paragraph to main section
- Info modal with title, question titles and answers. Also buttons to show answers
- Cool kino image 

### Kino Title
image
### Info modal
image
### Kino Image
image
### Short info
image
### Accessability
image
### Open times
image

## Checklist 
- [x] Title of cinema and a little information about opening times is shown on main page
- [x] Info modal is visible on main page and about page
- [x] When clicking any of the plus icons, a text appears and the plus icon changes to a minus
- [x] When clicking the minus icon the text dissapears and it turns into a plus icon again
- [x] Cool kino image is visible only on main page

### CODE REVIEW

File: pages/index/infoModal.js [hunk 1]
```diff
@@ -1,7 +1,96 @@
...
+    } else {
+      // Create elements
+      const question = document.createElement('p');
+      const answer = document.createElement('p');
+      const openBtn = document.createElement('img');
+      // Set class attribute to elements
+      question.setAttribute('class', 'modal-question');
+      answer.setAttribute('class', 'modal-answer');
+      openBtn.setAttribute('class', 'modal-open');
+      // Use JSON data to render text to elements
```
Comment (Emilyempa) [inline pages/index/infoModal.js:44]: Just a curious question...why did you choose to use setAttribute instead of .classList.add?

**Review by Emilyempa — APPROVED** [review]
Nice work, Viktor! It works as you described in the PR and as the PBI. I really liked that your PR was nicely written and that it was easy to follow all the steps

File: pages/index/infoModal.js (hunk 1 above)
Comment (viktornoskire) [reply pages/index/infoModal.js:44]: I just like the way you can add your own attribute, like "data-type='type'", so I've used this a little in my own projects but when you're pointing it out I'm realising that it's probably best practice to use classList.add. So if you want me to change it I can easily do that!

Comment (Emilyempa) [reply pages/index/infoModal.js:44]: No I was just curious, it works fine so you dont have to change it 👍

Comment (viktornoskire) [reply pages/index/infoModal.js:44]: Okay, then you can merge it! 😄

**Comment by 0x0xoscar** [comment]
Great PR especially that you included pictures :)

**Pull request merged**